transformers/events/tend/transformer_test.go

```diff
@@ -44,11 +44,18 @@ var _ = Describe("Tend transformer", func() {
 			models, err := transformer.ToModels(constants.FlipABI(), []core.EventLog{test_data.TendEventLog}, db)
 			Expect(err).NotTo(HaveOccurred())
 
-			var addressID int64
+			var addressID, msgSenderAddressID int64
+
 			addrErr := db.Get(&addressID, `SELECT id FROM public.addresses`)
 			Expect(addrErr).NotTo(HaveOccurred())
 			expectedModel := test_data.TendModel()
 			expectedModel.ColumnValues[event.AddressFK] = addressID
+
+			msgSenderAddressErr := db.Get(&msgSenderAddressID, `SELECT id FROM addresses WHERE address = $1`,
```

Comment:
Noticed this while working on the deal transformer as well, but idk why we're doing raw DB queries here (and above for the `addressID`) - would be a fan of using `shared.GetOrCreateAddress` (esp since I think now the query for the `addressID` could be flaky unless the contract address == `msg.sender`